### DISCUSSION TITLE
"Comment on this file" tooltips are obscuring if the file is marked as viewed

### DISCUSSION THREAD

**Discussion by geeekgod** [community/community#74689] [Pull Requests]
### Select Topic Area

Bug

### Body

While reviewing a PR I stumbled across a bug in the Pull Requests section. Let's say if I marked a file as `viewed` and just hover over the `Comment on this file` button, the tooltip is being hidden under the `file-header`.

I thought might be a fluke for a second but noticed the same behaviour form past 2 days.

Here's how it's occurring:

https://github.com/community/community/assets/56874272/3d6d4df3-0546-4249-8b09-3528c16e430a

PS:
I've started reviewing code on Github recently not sure if it has been there from a long time or not.

Thanks!

**Comment by kauaneiras** [comment]
My God!  and truth.  This bug really exists.  I will comment and give an update as this is a very important discussion. 🤔🫨

**Comment by kauaneiras** [comment]
🚀🚀🚀
